repose-aggregator/components/services/rate-limiting-service/src/main/java/org/openrepose/core/services/ratelimit/LimitKey.java

```diff
@@ -45,12 +40,12 @@ public static String getLimitKey(String limitGroup, String limitId, Matcher uriM
         cacheIdBuffer.append(String.valueOf(limitGroup.hashCode()));
 
         // All cacheId's contain the unique limit Id
-        cacheIdBuffer.append(":" + String.valueOf(limitId.hashCode()));
+        cacheIdBuffer.append(":").append(String.valueOf(limitId.hashCode()));
```

Comment:
😢 🐼 

Indeed. This is one of those times where I see old code that I wrote and wish that I hadn't been the person to write it.